client/src/components/Companies/Company.js

```diff
@@ -76,7 +77,7 @@ const companyCit = [
     {
         RelationalID: "",
         Author: "",
-        Date: "",
+        Date: 0, /*changed datatype from string to int for listing citations chronologically*/
```

Comment:
Can't just change the type in frontend,  the backend should be changed accordingly.  How would you handle data like 'June 2020' which is real data from our database.